nuxeo-ai-core/src/main/resources/OSGI-INF/ai-web-ui-properties.xml

```diff
@@ -6,6 +6,9 @@
   <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
     <!-- enrichers -->
     <property name="org.nuxeo.web.ui.enrichers.document" list="true">aiModels</property>
+    <property name="org.nuxeo.web.ui.insight.client.enabled">
```

Comment:
Im kind of curious: i don't understand how this property is working actually `org.nuxeo.web.ui.insight.client.enabled` is not referenced in the `_isAvailable` method